apps/src/templates/studioHomepages/JoinSectionNotifications.jsx

```diff
@@ -29,6 +29,7 @@ const JoinSectionSuccessNotification = ({sectionName}) => (
     notice={i18n.sectionsNotificationSuccess()}
     details={i18n.sectionsNotificationJoinSuccess({sectionName})}
     dismissible={true}
+    isRtl={false}
```

Comment:
Yes, but they don't have access to that prop yet. I was going to hook it up to the store with the Redux work and switch to the real value then.